README.md

```diff
@@ -19,4 +19,29 @@ This script will create Toggl projects with the name `#shotgun_ticket_number sho
 
 This script will create Shotgun Time Logs on tickets for every project time log that exist in Toggl for the last 7 days. You can change the period of time using the `--start` and `-end` command line arguments.
 
+# toggl.rb
 
+## Before first use
+WITH YOUR **SYSTEM** RUBY (`/usr/local/bin/ruby` (for RVM: `rvm use system`)) run: `gem install bundler; bundle install`
```

Comment:
my system ruby is 2.4.0 and there's a bug in last version of awesome_print published in rubygem...